.github/workflows/pr-rebase-needed.yml

```
@@ -9,7 +9,7 @@ permissions:
   pull-requests: write # To add labels and comment on PRs
 
 jobs:
-  idle:
+  mergeable:
```

For consistency, see: https://github.com/search?q=org%3Amdn+label-rebase-needed&type=code

```suggestion
  label-rebase-needed:
```

---

Updated.

Went to check for any patterns unluckily only to https://github.com/mdn/content/blob/4578af853ec3e520f4f2038c028c265591cbaa70/.github/workflows/pr-rebase-needed.yml#L1-L18 inferring the wrong idea there (preference for shortest unique id) no matter how's the reusable itself identified.